src/components/frame/v5/pages/BalancePage/partials/BalanceTable/BalanceTable.tsx

```diff
@@ -1,124 +1,162 @@
-        }}
-        initialState={{
-          pagination: {
-            pageSize: 10,
-          },
-        }}
-        showPageNumber={data.length >= 10}
-        onSortingChange={setSorting}
-        onRowSelectionChange={setRowSelection}
-        getSortedRowModel={getSortedRowModel()}
-        getPaginationRowModel={getPaginationRowModel()}
-        emptyContent={
-          !tokensDataLength && (
-            <div className="border border-1 w-full rounded-b-lg border-gray-200">
-              <EmptyContent
-                icon="binoculars"
-                title={{ id: 'balancePage.table.emptyTitle' }}
-                description={{ id: 'balancePage.table.emptyDescription' }}
-                withoutButtonIcon
-              />
-            </div>
-          )
-        }
-        getMenuProps={getMenuProps}
-        renderCellWrapper={(className, content) => (
-          <div className={clsx(className, 'min-h-[3.625rem] !py-[0.1rem]')}>
-            {content}
-          </div>
-        )}
-      >
-        <>
-          {/* # TODO Enable correct filtering */}
-          {/* {(!!tokensDataLength || !!searchValue) && <Filter />} */}
+      <div className="flex sm:justify-between sm:items-center sm:flex-row flex-col pb-[.875rem]">
+        <h4 className="heading-5">
+          {formatText({ id: 'balancePage.table.title' })}
+        </h4>
+        <div className="flex items-center mt-2.5 sm:mt-0">
           <Button
             mode="primarySolid"
-            className="ml-2"
             onClick={toggleAddFundsModalOn}
             size="small"
           >
             {formatText({ id: 'balancePage.table.addFunds' })}
           </Button>
-        </>
-      </TableWithHeaderAndMeatballMenu>
+        </div>
+      </div>
+
+      {data.length === 0 ? (
+        <EmptyContent
+          className="border rounded-b-lg border-gray-200"
+          icon="binoculars"
+          title={{ id: 'balancePage.table.emptyTitle' }}
+          description={{ id: 'balancePage.table.emptyDescription' }}
+          withoutButtonIcon
+        />
+      ) : (
+        <div className="overflow-x-auto max-w-full">
+          <Table>
+            <tr>
+              <th>{formatText({ id: 'table.row.asset' })}</th>
+              <th className="hidden sm:table-cell">
+                {formatText({ id: 'table.row.symbol' })}
+              </th>
+              <th className="hidden sm:table-cell">
+                {formatText({ id: 'table.row.type' })}
+              </th>
+              <th>
+                <button
+                  type="button"
+                  className="flex items-center gap-1"
+                  onClick={() => {
+                    handleSortFieldClick(BalanceTableSortFields.BALANCE);
+                  }}
+                >
+                  {formatText({ id: 'table.row.balance' })}
+                  {getSortArrow(BalanceTableSortFields.BALANCE)}
+                </button>
+              </th>
+
+              <th> </th>
```

Comment:
Is this needed?

Reply:
yeah for proper formatting, I could maybe colspan the last header